dev/tools/controllerbuilder/pkg/commands/generatetypes/generatetypescommand.go

```diff
@@ -62,10 +66,22 @@ func BuildCommand(baseOptions *options.GenerateOptions) *cobra.Command {
 
 func RunGenerateCRD(ctx context.Context, o *GenerateCRDOptions) error {
 	if o.ServiceName == "" {
-		return fmt.Errorf("ServiceName is required")
+		return fmt.Errorf("`service` is required")
 	}
 	if o.GenerateOptions.ProtoSourcePath == "" {
-		return fmt.Errorf("ProtoSourcePath is required")
+		return fmt.Errorf("`proto-source-path` is required")
+	}
+
+	if o.KindNames != nil {
+		errMsg := []string{}
```

Comment:
Suggestion for the future; you can now do `var errs []error  ... if len(errs) != 0 { return errors.Join(errs...) } `